LIPS/lip-4.md

```diff
@@ -0,0 +1,47 @@
+---
+lip: 4
+title: 72h Aragon Votes
+status: WIP
+author: Victor Suzdalev (kadmil)
+discussions-to: https://research.lido.fi/t/increase-the-dao-voting-duration/1048
+created: 2021-09-28
+updated: -
+---
+
+# 72h Aragon Votes
+
+Currently Aragon votes on Lido DAO are 24h long. This timeframe has proved to be short enough for reaching the quorum even on non-contentios votes. While most routine DAO operations are expected to switch to EasyTracks (vehicle for pre-approved opt-out DAO actions), any custom blockchain action on behalf of DAO would still require the Aragon vote. The proposal is to make Aragon votes longer, allowing more LDO holders to participate.
+
+## Problem
+
+Gathering the quorum on Aragon votes within 24h window proved difficult enough even for repetitive and non-contentios actions such as raising Node Operators' keys limits and refilling liquidity pools' rewards programs.
+
+## Proposed solution
+
+The proposed solution is to make Aragon votes last 72h instead of the current 24h. To achieve this, the DAO would have to upgrade the Voting smart contract, as the current version doesn't allow changing the vote duration.
+
+The Voting contract version where the vote duration can be changed can be found [there](https://github.com/lidofinance/aragon-apps/blob/8c46da8704d0011c42ece2896dbf4aeee069b84a/apps/voting/contracts/Voting.sol)
```

Comment:
+audit link

Reply:
We wanted to share the LIP with auditors first, and will add it there after the audit is published